README.md

```diff
@@ -43,18 +37,9 @@ The code is compiled and the latest NuGet packages are located at
 /home/pi/localNuget
 ```
 
-You can activate the Python environment using the command:
-```sh
-pi@pi-top:~ source /home/pi/.jupyter_venv/bin/activate
-```
 
 ## Running Visual Studio Code Insiders and the .NET Interactive notebook extension on the device
 
-You can install VS Code Insiders on your pi-top:
-```sh
-pi@pi-top:~ wget https://update.code.visualstudio.com/latest/linux-deb-armhf/insider -O code-insiders_1.50.0-1601271790_armhf.deb
-pi@pi-top:~ sudo dpkg -i code-insiders_1.50.0-1601271790_armhf.deb
-```
 Then add the [.NET interactive extension](https://github.com/dotnet/interactive#visual-studio-code) and now you can use .NET Interactive notebooks directly on your pi-top.
```

Comment:
This doesn't make sense now. Does it still require VS Code Insiders? Should we provide a way to install it from apt repository?

Reply:
we do not need vscode insider, we are already releasing extensions for vscode stable